vector/src/main/res/values/strings.xml

```diff
@@ -3221,5 +3221,11 @@
     <string name="device_manager_other_sessions_view_all">View All (%1$d)</string>
     <string name="device_manager_other_sessions_description_verified">Verified · Last activity %1$s</string>
     <string name="device_manager_other_sessions_description_unverified">Unverified · Last activity %1$s</string>
+    <string name="device_manager_header_section_security_recommendations_title">Security recommendations</string>
+    <string name="device_manager_header_section_security_recommendations_description">Improve your account security by following these recommendations.</string>
+    <string name="device_manager_unverified_sessions_title">Unverified sessions</string>
+    <string name="device_manager_unverified_sessions_description">Verify or sign out from unverified sessions.</string>
+    <string name="device_manager_inactive_sessions_title">Inactive sessions</string>
+    <string name="device_manager_inactive_sessions_description">Consider signing out from old sessions (90 days or older) you don’t use anymore.</string>
```

Comment:
Is there a word missing here?

```suggestion
    <string name="device_manager_inactive_sessions_description">Consider signing out from old sessions (90 days or older) you don’t use them anymore.</string>
```

Also maybe `90 days or more` is better than `90 days or older`.

Disclaimer: I am not a native speaker!

Also 90 is hard-coded here, we could use `%d` and use `SESSION_IS_MARKED_AS_INACTIVE_AFTER_DAYS` for more flexibility (Element or their forks).

Reply:
Done. I believe the missing word was "that".